LiskHQ/lisk-hub
# Fix Nan for very small amount in 'Activity tab' - Closes #507

### What was the problem?
Numeral.js repository have issue with handling small amounts
### How did I fix it?
I Forked repo and implemented custom scientific notation to decimal
### How to test it?
Connect to testNet, find `3583097017580240276L` in explore, go inside `Outgoing` tab,
check if transaction amount from Mar 2, 2018 is `-0.00000045` 

### Review checklist
- The PR solves #507 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
